python/docs/migration/basics.md

```diff
@@ -1,52 +1,43 @@
-# v0 to v1 Basics
+# Basics - v0 to v1
 
-## Performance Improvement
+whylogs v1 has been released on **May 31st, 2022**. While this release represents an exciting milestone for the whyogs library, there are some important implications for current users of whylogs v0. Most notably, whylogs v1 comes with a simplified and more intuitive API. This means that if you choose to upgrade to v1, code changes will be required.
 
-* Native integration with
+Users can visit this [migration guide](https://docs.whylabs.ai/docs/v1-migration/) to assist with migrating their code to whylogs v1.
 
-### Columnar Operations
```

Comment:
I removed the sections about Columnar Operations for now. If someone could provide me with one or two phrases about this, I can update it.